src/app/feedback-form/feedback-form.component.html

```diff
@@ -1,54 +1,75 @@
 <section class="contact-form">
-  <form action="" class="contact-form__container">
+  <form [formGroup]="form" action="" class="contact-form__container">
     <fieldset class="contact-form__fieldset">
       <header class="contact-form__header">
         <h3 class="contact-form__title">Get in Touch with Us!</h3>
         <p class="contact-form__subtitle">We'd love to hear from you. Please fill out the form below.</p>
       </header>
 
       <div class="contact-form__form-group">
-        <label class="contact-form__label" for="name">Your Name:</label>
-        <input placeholder="Enter your name" class="contact-form__input" type="text" id="name" name="name" required />
+        <p-floatLabel>
+          <input class="contact-form__input" type="text" id="name" formControlName="name" pInputTextarea />
```

Comment:
this needs to be text not textarea

Reply:
In field 'Enter name" I use approach with pInputTextarea